### PR TITLE
Moving to hmcts version of liquibase which performs better DB locking

### PR DESCRIPTION
**Pull request by adusumillipraveen**
This based on https://github.com/hmcts/liquibase/pull/2

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by smathangi** [comment]
Moved to testcontainers from embedded postgres (otj-pg-embedded). 
somehow otj-pg-embedded is not compatible with latest version of liquibase-core  as it fails to create sequence for column type "serial" . Similar to the problem mentioned here https://liquibase.jira.com/browse/CORE-1816. The error occurs against otj-pg-embedded only. 

Eg: `Caused by: org.postgresql.util.PSQLException: ERROR: null value in column "id" violates not-null constraint
  Detail: Failing row contains (null, TEST2, user@hmcts.net, TEST, workBasketDefaultCaseType, workBasketDefaultState, 2019-11-18 23:17:04.184981, UPDATE, rtaashrie9j1otx@example.com).`

File: src/test/java/uk/gov/hmcts/ccd/BaseTest.java [hunk 1]
```diff
@@ -24,12 +23,11 @@
...
 
     protected static final MediaType contentType = new MediaType(MediaType.APPLICATION_JSON.getType(),
-                                                                 MediaType.APPLICATION_JSON.getSubtype(),
-                                                                 Charset.forName("utf8"));
+        MediaType.APPLICATION_JSON.getSubtype(),
+        Charset.forName("utf8"));
     protected static final ObjectMapper mapper = new ObjectMapper();
 
     @Autowired
-    @Qualifier("EmbeddedPostgres")
```
Comment (mario-paniccia) [inline src/test/java/uk/gov/hmcts/ccd/BaseTest.java]: Hi Praveen what's the motivation for this change please?

File: src/test/java/uk/gov/hmcts/ccd/data/TestConfiguration.java [hunk 2]
```diff
@@ -1,34 +0,0 @@
-package uk.gov.hmcts.ccd.data;
-
-import com.opentable.db.postgres.embedded.EmbeddedPostgres;
```
Comment (mario-paniccia) [inline src/test/java/uk/gov/hmcts/ccd/data/TestConfiguration.java]: Hi Praveen what's the motivation for this change please?

File: src/test/java/uk/gov/hmcts/ccd/BaseTest.java (hunk 1 above)
Comment (timja) [reply src/test/java/uk/gov/hmcts/ccd/BaseTest.java]: @mario-paniccia Sateesh added it, see here: https://github.com/hmcts/ccd-user-profile-api/pull/217#issuecomment-556345189

**Comment by mario-paniccia** [comment]
> Moved to testcontainers from embedded postgres (otj-pg-embedded).
> somehow otj-pg-embedded is not compatible with latest version of liquibase-core as it fails to create sequence for column type "serial" . Similar to the problem mentioned here https://liquibase.jira.com/browse/CORE-1816. The error occurs against otj-pg-embedded only.
> 
> Eg: `Caused by: org.postgresql.util.PSQLException: ERROR: null value in column "id" violates not-null constraint Detail: Failing row contains (null, TEST2, user@hmcts.net, TEST, workBasketDefaultCaseType, workBasketDefaultState, 2019-11-18 23:17:04.184981, UPDATE, rtaashrie9j1otx@example.com).`

👍
